CASMINST-5067 (#2208)
Cray-HPE/docs-csm: troubleshooting/known_issues/check_system_management_monitoring_tools.md

> # 1.3 Check of system management monitoring tools

Reviewer: ```suggestion
# Check of system management monitoring tools
```

If all designated prerequisites are met, the availability of system management health services may optionally be validated by accessing the URLs listed in
[Access System Management Health Services](../../operations/system_management_health/Access_System_Management_Health_Services.md).

> It is very important to check the `Prerequisites` section of this document.

Reviewer: Prerequisites section of which document? There isn't one in this document that I can see. If it's in the linked document above, then I recommend linking directly to that section here, to prevent confusion about which document is being referenced.

> It is very important to check the `Prerequisites` section of this document.

Reviewer 2: ```suggestion
It is recommended to review the [Prerequisites](../../operations/system_management_health/Access_System_Management_Health_Services.md#prerequisites) content.
```

> If one or more of the the URLs listed in the procedure are inaccessible, it does not necessarily mean that system is not healthy. It may simply mean that not all of the

Reviewer 2: ```suggestion
If any of the URLs that are listed in the procedure are inaccessible, it does not reflect that the system is unhealthy. It is indicating that not all of the prerequisites are met to allow access to the system management health tools through URL.
```

> prerequisites have been met to allow access to the system management health tools via URL.

Reviewer 2: ```suggestion
```

> Information to assist with troubleshooting some of the components mentioned in the prerequisites can be accessed here:

Reviewer 2: ```suggestion
The following references assist with troubleshooting some of the components mentioned in the prerequisites:
```
